packages/cli/src/version.test.ts

```diff
@@ -1,13 +1,24 @@
 import fs from 'fs'
 import path from 'path'
 
-const scriptPath = path.join(__dirname, 'index.ts')
-import { afterAll, afterEach, beforeAll, describe, expect, it, jest } from '@jest/globals'
-import * as monoweave from '@monoweave/node'
+import {
+    afterAll,
+    afterEach,
+    beforeAll,
+    describe,
+    expect,
+    it,
+    jest as jestImport,
+} from '@jest/globals'
 import { createTempDir, waitFor } from '@monoweave/test-utils'
 import { parseDeferredVersionFile } from '@monoweave/versions'
 
-jest.mock('@monoweave/node')
+const scriptPath = path.join(__dirname, 'index.ts')
+
+// Skipping the git mock as we use a temp repository for these tests.
+// @ts-expect-error https://github.com/swc-project/plugins/issues/310
+// eslint-disable-next-line @typescript-eslint/consistent-type-imports, import-x/newline-after-import
+;(jest as typeof import('@jest/globals').jest).mock('@monoweave/node')
```

Comment:
temporary until we switch to ESM https://github.com/swc-project/plugins/issues/310